MAUI/Autocomplete/UI-Customization.md

```diff
@@ -120,6 +120,30 @@ The following gif image illustrates the result of the above code.
 
 ![.NET MAUI Autocomplete border color](Images/UICustomization/BorderColor.png)
 
+## Customize the selection text highlightColor 
+
+The SelectionTextHighlightColor property is used to modify the background color of selected item text in the autocomplete control.
```

Comment:
The SelectionTextHighlightColor property is used to modify the background color of the selected item text in the autocomplete control.